Pass the correct number of arguments to 'nick_strip' and 'nick_ok' in tests.

Patch by Mark H Weaver <mhw@netris.org>.

--- bitlbee/tests/check_nick.c.orig	2013-11-27 17:54:54.000000000 -0500
+++ bitlbee/tests/check_nick.c	2014-03-05 23:41:45.761230468 -0500
@@ -30,7 +30,7 @@ START_TEST(test_nick_strip)
 	for (i = 0; get[i]; i++) {
 		char copy[60];
 		strcpy(copy, get[i]);
-		nick_strip(copy);
+		nick_strip(NULL, copy);
 		fail_unless (strcmp(copy, expected[i]) == 0, 
 					 "(%d) nick_strip broken: %s -> %s (expected: %s)", 
 					 i, get[i], copy, expected[i]);
@@ -45,7 +45,7 @@ START_TEST(test_nick_ok_ok)
 	int i;
 
 	for (i = 0; nicks[i]; i++) {
-		fail_unless (nick_ok(nicks[i]) == 1,
+		fail_unless (nick_ok(NULL, nicks[i]) == 1,
 					 "nick_ok() failed: %s", nicks[i]);
 	}
 }
@@ -58,7 +58,7 @@ START_TEST(test_nick_ok_notok)
 	int i;
 
 	for (i = 0; nicks[i]; i++) {
-		fail_unless (nick_ok(nicks[i]) == 0,
+		fail_unless (nick_ok(NULL, nicks[i]) == 0,
 					 "nick_ok() succeeded for invalid: %s", nicks[i]);
 	}
 }
